### PR TITLE
[WIP] Hindi Translit

### PR DESCRIPTION
**Pull request by bogas04**
* [x] Hindi unicode keeps "i" after the letter, while in gurakhar it's before the letter. 
* [x] Other maatras.

### CODE REVIEW

File: src/translit_modules/hindi.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    if (gurmukhiLetter === 'i') {
+      str = str.replace(/i./gm, full =>
+        full
+          .split('')
+          .reverse()
+          .join('')
+      );
+    }
+
+    while (str.includes(gurmukhiLetter)) {
```
Comment (bogas04) [inline src/translit_modules/hindi.js]: IDK how to do this via RegExp in a sane way lol.

**Comment by ManjotS** [comment]
@bogas04 what is status of this?

**Comment by bogas04** [comment]
There are some missing cases but I guess we can merge it as a beta to dogfood and fix those. That'll be much faster.
I'll resolve the conflicts and prepare the PR

**Comment by ManjotS** [comment]
letters missing
``` ['&', ''], // phapha pair bindi
  ['@', ''], // halant
  ['E', ''], // open oora
  ['H', ''], // pair haha
  ['L', ''], // lala pair bindi
  ['N', ''], // tippee
  ['Z', ''], // gaga pair bindi
  ['^', ''], // khakha pair bindi
  ['`', ''], // adhak over letter
  ['z', ''], // jaja pair bindi zaza
  ['~', ''], // adhak after letter
  ['¤', ''], // adhak after letter
  ['´', ''], // yakash (pair yaya)
  ['µ', 'ं'], // bindi
  ['Í', ''], // pair vava
  ['Î', ''], // half yaya
  ['Ø', ''], // extra top line (extender)
  ['ç', ''], // pair chacha
  ['œ', ''], // pair tata
  ['ŧ', ''], // bad char
  ['˜', ''], // pair nana
  ['†', ''], // pair tanka```

**Comment by bogas04** [comment]
```
 ['@', ''], // halant
  ['E', 'ऄ'], // open oora
  ['H', 'ह्'], // pair haha
  ['L', 'ऴ'], // lala pair bindi
  ['N', 'ं'], // tippee
  ['Z', 'ग़'], // gaga pair bindi
  ['^', 'ख़'], // khakha pair bindi
  ['`', 'ँ'], // adhak over letter
  ['z', 'ज़'], // jaja pair bindi zaza
  ['~', ''ँ'], // adhak after letter
  ['¤', ''ँ'], // adhak after letter
  ['´', 'य्'], // yakash (pair yaya)
  ['µ', 'ं'], // bindi
  ['Í', 'व्'], // pair vava
  ['Î', 'य्'], // half yaya
  ['Ø', ''], // extra top line (extender)
  ['ç', 'च्'], // pair chacha
  ['œ', 'त्'], // pair tata
  ['ŧ', ''], // bad char
  ['˜', 'न्'], // pair nana
  ['†', 'ट्'], // pair tanka
```

**Comment by tarunsingh5** [comment]
whats the status here? can we soft launch?

**Comment by bogas04** [comment]
Yes I want a soft launch only coz fixing bugs like this is bit hard. We can make it a beta release and let our users tell about the mistakes in transliteration. 

@tarunsingh5 @navdeepsinghkhalsa

File: package.json [hunk 2]
```diff
@@ -50,5 +50,12 @@
       "jest": true,
       "node": true
     }
+  },
+  "jest": {
+    "testURL": "http://localhost"
+  },
+  "prettier": {
```
Comment (navdeepsinghkhalsa) [inline package.json]: Could you please remove `prettier`? Our ESLint configuration is enough for 'opinionated' formatting :)

**Review by navdeepsinghkhalsa — CHANGES_REQUESTED** [review]
Could you please remove prettier and fix the linting errors?

File: package.json (hunk 2 above)
Comment (bogas04) [reply package.json]: done!

**Review by navdeepsinghkhalsa — APPROVED** [review]

**Pull request merged**